test/controllers/test-logout.js

```diff
@@ -52,4 +52,26 @@ describe('logout', function() {
       .expect('Set-Cookie', /refresh_token=;/)
       .end(done);
   });
+
+  describe('when Accept header is set to text/html', function() {
```

Comment:
You're right. My bad. I was just used to lowercase being used as the general style, but I can see now that it's being used differently.